test/monitoring/specs/aws.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "aws": "true"
+  },
+  "output": [
+    "setup",
+    "--url",
+    "https://adobeioruntime.net/api/v1/web///ops@v2/_status_check/healthcheck.json",
+    "--url",
+    "https://.execute-api..amazonaws.com/helix-services/word2md/v2/_status_check/healthcheck.json",
+    "--name",
+    "--name",
+    "ops.aws"
+  ]
```

Comment:
Setting `parameter.aws` to `true` leads to a double `--name` option, which might be wrong...

Reply:
I think this could only happens if the default values of unspecified parameters are missing. The first URL also lacks a few values :)

Reply:
The defaults are specified in the orb command config. I will add them to the test specs.